skin/src/main/java/org/researchstack/skin/AppPrefs.java

```diff
@@ -24,11 +24,16 @@
         prefs = PreferenceManager.getDefaultSharedPreferences(context);
     }
 
-    public static synchronized AppPrefs getInstance(Context context)
+    public static void init(Context context) {
+        instance = new AppPrefs(context);
+    }
+
+    public static synchronized AppPrefs getInstance()
```

Comment:
Do we want to keep this class backwards compatible and mark the old one as `@Deprecated`?

I do like this change -- seems like a bad idea to require context to be passed every time since most of the time the context is ignored. 

Or, if the goal is to simply force loading of AppPrefs at a certain point, we could call AppPrefs.getInstance(context) in ResearchStack.init